integration-tests/emulated/tests/bridges/bridge-hub-polkadot/src/tests/mod.rs

```diff
@@ -73,7 +74,7 @@ pub(crate) fn weth_at_asset_hubs() -> xcm::v4::Location {
 			xcm::v4::Junction::GlobalConsensus(xcm::v4::NetworkId::Ethereum {
 				chain_id: snowbridge::CHAIN_ID,
 			}),
-			xcm::v4::Junction::AccountKey20 { network: None, key: snowbridge::WETH },
+			xcm::v4::Junction::AccountKey20 { network: None, key: WETH },
 		],
 	)
```

Comment:
If we move 

```
xcm::v4::Location::new(
						2,
						[
							xcm::v4::Junction::GlobalConsensus(EthereumNetworkXcmV4::get()),
							xcm::v4::Junction::AccountKey20 { network: None, key: WETH },
						],
					),
```

into common, we can remove this duplicate as well.

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/625/commits/748af5085e2289813ea502301c4c7173e33633ae.